packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/upgrade-request/upgrade-request.controller.js

```diff
@@ -0,0 +1,73 @@
+import get from 'lodash/get';
+import { buildURL } from '@ovh-ux/ufrontend/url-builder';
+import { ISSUE_TYPE_ID } from './upgrade.constants';
+
+export default class DedicatedUpgradeController {
+  /* @ngInject */
+  constructor($translate, OvhApiSupport, atInternet, Alerter) {
+    this.$translate = $translate;
+    this.OvhApiSupport = OvhApiSupport;
+    this.atInternet = atInternet;
+    this.Alerter = Alerter;
+  }
+
+  $onInit() {
+    this.isLoading = false;
+  }
+
+  cancel() {
+    this.atInternet.trackClick({
+      name: `dedicated::dedicated::server::dashboard::upgrade::${this.selectedUpgrade}::cancel`,
```

Comment:
The name of the hit could be set in a prefix variable (in the initialisation or elsewhere)